ogre2/src/Ogre2RenderEngine.cc

```diff
@@ -415,7 +415,6 @@ void Ogre2RenderEngine::CreateLogger()
 //////////////////////////////////////////////////
 void Ogre2RenderEngine::CreateContext()
 {
-#if !defined(__APPLE__) && !defined(_WIN32)
```

Comment:
This was removed in #646 back in June, but it hadn't been merged forward. Is what I did the correct thing to do?

Reply:
Yes, this looks correct to me :+1: